static/app/views/issueList/addViewPage.tsx

```diff
@@ -97,18 +98,55 @@ function SearchSuggestionList({
   searchSuggestions,
   type,
 }: SearchSuggestionListProps) {
-  const {onNewViewSaved} = useContext(NewTabContext);
+  const {onNewViewsSaved} = useContext(NewTabContext);
   const organization = useOrganization();
 
   return (
     <Suggestions>
-      <TitleWrapper>{title}</TitleWrapper>
+      <TitleWrapper>
+        {title}
+        {type === 'saved_searches' && (
+          <StyledButton
+            size="zero"
+            onClick={e => {
+              e.stopPropagation();
+              openConfirmModal({
+                message: (
+                  <b style={{display: 'flex', justifyContent: 'center'}}>
```

Comment:
Can this be a styled component? And usually you don't want a `b` element be block level, you usually have a wrapper div or p tag